xbmc-addon-nrk-master/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.nrk"
+       name="NRK Nett-TV"
+       version="4.7.99"
+       provider-name="takoi+a99b">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.requests" version="1.0.4"/>
+    <import addon="script.module.routing" version="0.1.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="main.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <language>no</language>
+    <summary lang="en">Watch NRK Web TV</summary>
+    <summary lang="no">Se NRK Nett-TV</summary>
+    <summary lang="it">Guarda NRK (in Norvegia)</summary>
+    <description lang="en">NRK Web TV gives you the opportunity to watch live TV as well as recordings of many shows.</description>
+    <description lang="no">NRK Nett-TV gir deg muligheten til å se mange av NRKs programmer både direkte og i opptak.</description>
+    <description lang="it">Questo addon per NRK rende possibile la visione di molti programmi di NRK sia in diretta che in differita.</description>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
+    <forum>https://forum.kodi.tv/showthread.php?tid=52824</forum>
+    <source>https://github.com/tamland/xbmc-addon-nrk</source>
```

Comment:
Is @tamland still the maintainer? Because these sources are not in his repo?

Reply:
Hi, i don't know - he seems to be absent for some time, it was a bit awkward to see Nexus without NRK, so i'm trying to fill in. If he gets back to it he can also get my mods if he like them, and add to his main, no issue on my side. I did a 0.1% to just make it work under Nexus. All the work is his, obviously.

Reply:
I'm not actively maintaining it and have left it up to others/the community. I don't know if those people are also inactive now or not, however there appears to be activity in the repo 2 week ago? https://github.com/tamland/xbmc-addon-nrk/commits/master

Reply:
So would it make sense to do this PR there and and then a proper PR from [https://github.com/tamland/xbmc-addon-nrk](https://github.com/tamland/xbmc-addon-nrk/commits/master) to Kodi?

Reply:
I have merged the open PR at least https://github.com/tamland/xbmc-addon-nrk/pull/73

Reply:
happy to have helped